src/components/CountryPerformanceOnRisk.js

```diff
@@ -1,126 +1,153 @@
 import React, { Component } from 'react';
-import PlotlyGraph from './Plot.js'
+import { connect } from 'react-redux'
+import PlotlyGraph from './Plot.js';
+import Select from 'react-select';
+import 'react-select/dist/react-select.css';
 
 
-class CountryPerformanceOnRisk extends Component {
+export class CountryPerformanceOnRisk extends Component {
   constructor(props) {
     super(props)
     this.state = {
       data: [],
       graphOptions: {},
       countries: [],
-      matchedCountry: {},
-      graphsToShow: [],
-      graphs: {}
+      defaultCountry: {},
+      selected1: undefined,
+      selected2: undefined,
+      selected3: undefined
     }
+    this.updateValue1 = this.updateValue1.bind(this)
+    this.updateValue2 = this.updateValue2.bind(this)
+    this.updateValue3 = this.updateValue3.bind(this)
   }
```

Comment:
@zelima when there will be multiple graphs they will be instantiated from different containers in parallel, so they are independent of each other.
To be able to have a single function we need to pass some indicator from onChange event - which must be passed from the container as our CountrySelect component is generic. I think in the end we will have more not DRY code than we have now.